_digest/october-2020/an-effective-guide-to-bullet-journalling.mdx

```diff
@@ -0,0 +1,43 @@
+    <ol start="3"><li><a href="https://www.amazon.ca/Leuchtturm1917-Notebooks-Hardcover-Medium-Powder/dp/B07ZXJDY1Y/ref=sr_1_3_sspa?dchild=1&amp;keywords=Leuchtturm1917+Medium+A5+Notebook&amp;qid=1601422379&amp;s=office&amp;sr=1-3-spons&amp;psc=1&amp;spLa=ZW5jcnlwdGVkUXVhbGlmaWVyPUEyMjZCOEZBMVZEMEFFJmVuY3J5cHRlZElkPUEwNzEyMDI5MjRESjNGV0ZTS0FaQyZlbmNyeXB0ZWRBZElkPUEwNDA1NjYxMTNPS0FRVTlOSVZOVCZ3aWRnZXROYW1lPXNwX2F0ZiZhY3Rpb249Y2xpY2tSZWRpcmVjdCZkb05vdExvZ0NsaWNrPXRydWU="><strong>Leuchtturm1917 Medium A5 Notebook </strong></a><strong>($24.95 CAD)</strong></li></ol>
+    <p>I recommend Leuchttrum ONLY if you are 100% dedicated to bullet journaling. The paper quality and the aesthetic of the overall journal is the best of the best, although it is one of the pricier options. Several other notebooks on the market can easily replace this one if you do not require a bullet journal for excessively creative purposes (the Lemon book would easily serve the same purpose).</p>
+    <p>These bullet journaling books can conveniently be found on Amazon in accordance to your needs and what you want from a bullet journal.</p>
+    <p>Now, to jump into the basics of bullet journaling: I recommend starting off with discovering the different spreads many students use to organize their life. To this day, although I have outgrown a few, I still use many due to their effectiveness. This takes me to my next point: Please do not hesitate to discontinue spreads if they are overwhelming or hard to keep up with. I see a lot of people discontinue bullet journaling itself due to how hard it is to keep up with, but looking further into it, I realise that they think that they are <strong>supposed</strong> to include a certain number of spreads to effectively bullet journal. I can attest to this worry and say that I used to think that way, but it is in no way true. You are in control of how you want to plan your life, and that is the beauty that lies within bullet journaling. Whether you are in the mood for minimalist/materialist/artistic spreads, the key goal for bullet journaling is to channel your inner creativity and supply a source of organization in your life!&nbsp; Keep in mind: in no way do you have to make a complicated bullet journal spread to feel like you are journaling effectively! It is similar to any other creative outlet where the results are different for everyone. You need to assess what type of bullet journaling you are the best with, and make spreads accordingly! With that in mind, let me introduce you to the different spreads I use, and recommend my fellow students to use.</p>
+    <p><strong>Student&nbsp; Layouts</strong></p>
+    <p><strong>Semester layout</strong></p>
+    <figure class="wp-block-image"><img src="https://lh4.googleusercontent.com/Q8OxSB7i2MZBQfmEm-AyeyIg1k24XnzjC17jqrXH6_MDIlLqxFDRq1a6DzldWBEwPn0-RgbPDqi3BnIhitzLkCNjcZ_r2q3CUrzEIUqyAz3aU0f3Hu7JTL5Le8AH4vgPSoqMsEWp" alt=""></figure>
```

Comment:
Remove the attributes that are `class=wp-*` and I think the images should be downloaded and put into the public/blog folder (sorry, forgot to mention this in our meeting)